src/main/groovy/io/openliberty/tools/gradle/Liberty.groovy

```diff
@@ -169,7 +169,11 @@ class Liberty implements Plugin<Project> {
     }
 
     private static File getInstallDir(Project project) {
-        if (project.liberty.installDir == null) {
+        if (project.hasProperty('liberty.installDir')) {
```

Comment:
Two things...I think we should call `checkAndAppendWlp(project)` here...and that method needs updating to look for the property before using `project.liberty.installDir`.  Or make a second `checkAndAppendWlp(String)` and pass in the path to check.